src/main_data.cpp

```diff
@@ -79,7 +79,7 @@ void Main_Data::Init() {
 			// first set to current directory for all platforms
 			project_path = ".";
 
-#if defined(GEKKO) || defined(SWITCH) || defined(__MORPHOS__) || defined(__amigaos4__) || defined(__AROS__) 
+#if defined(GEKKO) || defined(__SWITCH__) || defined(__MORPHOS__) || defined(__amigaos4__) || defined(__AROS__)
```

Comment:
I need to check this one on hardware.
It should not be needed since the filesystem overhaul in libnx, otherwise we would have bug reports by now. Current master works, but will compare the logged paths.